src/Main.js

```diff
@@ -1,18 +1,44 @@
 import React from "react";
-
-
 import HornedBeast from "./HornedBeast.js";
-
+import Form from 'react-bootstrap/Form';
+import Data from './data.json';
```

Comment:
Super awesome job on the lab, it is looking amazing 💯 

- Feedback here is for line 4, we should update this import to be lowercase. `data` vs `Data` since uppercase imports are reserved for components (classes) 👍